RankensteinApp/App.js

```diff
@@ -0,0 +1,183 @@
+import React, { Component } from 'react';
+import { StyleSheet, Text, TextInput, View, TouchableOpacity, Platform,} from 'react-native';
+import {createStackNavigator,} from 'react-navigation';
+
+
+
+class LoginScreen extends React.Component{
+  static navigationOptions = {
+    header: null
+  }
+  state = {
+      username: '',
+      password: '',
+	  titleText: "Rankenstein",
+  }
+  handleUsername = (text) => {
+      this.setState({ username: text })
+  }
+  handlePassword = (text) => {
+      this.setState({ password: text })
+  }
+  login = (username, pass) => {
+      alert('username: ' + username + ' password: ' + pass)
+  }
+  render() {
+    return (
+      <View style={styles.container}>
+		<Text style={styles.titleText} onPress={this.onPressTitle}>
+          {'\n'}{this.state.titleText}{'\n'}{'\n'}
+        </Text>
+		<TextInput style = {styles.input}
+           underlineColorAndroid = "transparent"
+           placeholder = "Username"
+           placeholderTextColor = "#fff"
+           autoCapitalize = "none"
+           onChangeText = {this.handleUsername}/>
+		<TextInput style = {styles.input}
+		   secureTextEntry={true}
+           underlineColorAndroid = "transparent"
+           placeholder = "Password"
+           placeholderTextColor = "#fff"
+           autoCapitalize = "none"
+           onChangeText = {this.handlePassword}/>
+		<TouchableOpacity
+               style = {styles.submitButton}
+               onPress = {
+                  () => this.login(this.state.username, this.state.password)
+               }>
+               <Text style = {styles.submitButtonText}> Login </Text>
+        </TouchableOpacity>
+		<TouchableOpacity
+               style = {styles.submitButton}
+               onPress = {
+                  () => this.props.navigation.navigate('Create',{title: 'WHATEVER'})
+               }>
+               <Text style = {styles.submitButtonText}> Create New Account </Text>
+        </TouchableOpacity>
+      </View>
+    );
+  }
+}
+
+class HomeScreen extends React.Component {
+	render() {
+    return (
+      <View style={{ flex: 1, alignItems: 'center', justifyContent: 'center' }}>
+        <Text>Details Screen</Text>
+      </View>
+    );
+  }
+}
+
+class CreateAccountScreen extends React.Component {
+	state = {
+      username: '',
+      password: '',
+	  verifypass: '',
```

Comment:
Same here. It looks like you're using tabs instead of spaces in some places - spaces are generally preferred in the industry since tabs can be treated variably based on the chosen text editor. You can configure gedit to use 4 spaces instead of tabs each time you press Tab.